<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KETI_SERVERClass</class>
 <widget class="QMainWindow" name="KETI_SERVERClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KETI_SERVER</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_stream">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>70</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>684</x>
      <y>250</y>
      <width>431</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="playerControlLayout">
       <item>
        <widget class="QPushButton" name="PlayButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StopButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentTimeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="TimeSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalTimeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="LoopCheckBox">
         <property name="text">
          <string>Loop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="saveOriginalCheckBox">
         <property name="text">
          <string>Save Original</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="saveHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="hideSettingsButton">
         <property name="text">
          <string>Hide Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1111</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="frameLayout_main">
     <item>
      <widget class="FrameLabel" name="frameLabel_main">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>285</y>
      <width>663</width>
      <height>179</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="InfoTab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="title">
      <string>Video Information</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="3">
       <widget class="QLabel" name="nFramesCapturedLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_23">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Processing Rate:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="processingRateLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="cameraResolutionLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="deviceNumberLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_26">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Resolution:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_27">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Framerate:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="captureRateLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="nFramesProcessedLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_28">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ROI:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="roiLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_22">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cursor:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="mouseCursorPosLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_24">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Filename:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Framenumber:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="currentFrameNumberLabel">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QProgressBar" name="saveProgressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>663</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1238</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE_OPEN">
    <property name="title">
     <string>FILE OPEN</string>
    </property>
    <addaction name="actionTOP"/>
    <addaction name="actionOPEN_BOTTOM"/>
    <addaction name="actionOPEN_LEFT"/>
    <addaction name="actionOPEN_RIGHT"/>
    <addaction name="actionOPEN_FRONT"/>
    <addaction name="actionOPEN_BACK"/>
    <addaction name="separator"/>
    <addaction name="actionEXIT"/>
   </widget>
   <widget class="QMenu" name="menuCodec">
    <property name="title">
     <string>Codec</string>
    </property>
    <addaction name="actionHEVC"/>
    <addaction name="actionMpeg_4"/>
    <addaction name="actionAV1"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFILE_OPEN"/>
   <addaction name="menuCodec"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTOP">
   <property name="text">
    <string>OPEN TOP</string>
   </property>
  </action>
  <action name="actionOPEN_BOTTOM">
   <property name="text">
    <string>OPEN BOTTOM</string>
   </property>
  </action>
  <action name="actionOPEN_LEFT">
   <property name="text">
    <string>OPEN LEFT</string>
   </property>
  </action>
  <action name="actionOPEN_RIGHT">
   <property name="text">
    <string>OPEN RIGHT</string>
   </property>
  </action>
  <action name="actionOPEN_FRONT">
   <property name="text">
    <string>OPEN FRONT</string>
   </property>
  </action>
  <action name="actionOPEN_BACK">
   <property name="text">
    <string>OPEN BACK</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
  <action name="actionHEVC">
   <property name="text">
    <string>HEVC</string>
   </property>
  </action>
  <action name="actionMpeg_4">
   <property name="text">
    <string>Mpeg-4</string>
   </property>
  </action>
  <action name="actionAV1">
   <property name="text">
    <string>AV1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FrameLabel</class>
   <extends>QLabel</extends>
   <header>FrameLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="KETI_SERVER.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>KETI_SERVERClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1170</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEXIT</sender>
   <signal>toggled(bool)</signal>
   <receiver>KETI_SERVERClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
